cmd/fabric-ca-server/config.go

```diff
@@ -578,6 +579,20 @@ func (s *ServerCmd) configInit() (err error) {
 		return err
 	}
 
+	// Info logging for CA config, includes yaml file settings and environment variable overrides
+	allSettings := s.myViper.AllSettings()
+	settingsYaml, err := yaml.Marshal(allSettings)
+	if err == nil {
+		log.Infof("CA config with combined yaml file and environment variable overrides:\n%s", settingsYaml)
```

Comment:
It seems the configuration can contain passwords, which is failing an FVT test that the log doesn't leak passwords

Reply:
Ok, I've filtered the config of sensitive configurations now.